scss/login.scss

```diff
@@ -0,0 +1,84 @@
+.background-login {
+  align-items: center;
+  background-color: $white-secondary;
+  display: flex;
+  flex-direction: column;
+  height: $total-height;
+  justify-content: center;
+	width: $total-width;
+}
+
+.container-login {
+	height: 400px;
+  width: 300px;
+  background-color: $white-primary;
+	border-top: 8px solid $blue;
+	display: flex;
+	flex-direction: column;
+
+	.login-logo {
+		height: 70px;
+		width: 212px;
+		margin: 20px 44px 0;
+	}
+	
+
+	.form-group {
+		border-bottom: 3px solid $grey;
+		padding-bottom: 24px;
+		width: 252px;
+		margin: 0 24px;
+		justify-content: center;
+
+		label {
+			color: $black-default;
+			font-size: 16px;
+			height: 22px;
+			line-height: 22px;
+			margin: 16px 0 8px;
+		}
+	
+		input {
+			height: 32px;
+			width: 252px;
+			border-radius: 5px;
+			background-color: $white-secondary;
+		}
+	}
+	
+	.btn {
+		height: 32px;
+		width: 252px;
+		text-align: center;
+		border-radius: 5px;
+
+		.btn-green {
```

Comment:
If `.btn-green` extends .btn it's not necessary to nest the classes. They can be defined like this:
```scss
.btn {
  ...
}

.btn-green {
  @extend .btn;
  ...
}
```